src/VkBootstrap.h

```diff
@@ -69,6 +69,20 @@
 #define VKB_VK_API_VERSION_1_0 VKB_MAKE_VK_VERSION(0, 1, 0, 0)
 #endif
 
+// If compiled as a shared library, symbols need to be exported
+
+#if !defined(VK_SHARED)
+#define VK_API
```

Comment:
Going to be that guy and suggest we rename it to VKB_EXPORT to remove confusion of who is defining the macro and prevent possibly collisions.